example_function2/function_config.yaml

```diff
@@ -1,3 +1,6 @@
+# Note: Parameters can be hardcoded directly in 'deploy-push.yaml'
+# if you do not need to vary them between your functions, for
+# example 'cdf_project' or 'cdf_cluster'.
 cdf_project: forge-sandbox
 cdf_cluster: greenfield  # or api, westeurope-1, etc.
```

Comment:
Should default be greenfield?

Reply:
No 😜 This repo deploys on merges to master, so the parameters are real!